auth0/src/main/java/com/auth0/android/authentication/storage/SecureCredentialsManager.kt

```diff
@@ -303,9 +302,21 @@ public class SecureCredentialsManager @VisibleForTesting(otherwise = VisibleForT
             decryptCallback = null
             return
         }
-        val credentials = gson.fromJson(json, Credentials::class.java)
+        val bridgeCredentials = gson.fromJson(json, OptionalCredentials::class.java)
```

Comment:
an internal class `OptionalCredentials` was introduced and is only used on this line. It helps to parse credentials JSON that could have null values on it.